docker/Dockerfile

```diff
@@ -71,7 +71,9 @@ RUN $YARN build --filter=backend && \
   # Already imported the packages above; need to `yarn install` on the `dist-dynamic` sub-folder for backend plugins
   $YARN --cwd ./dynamic-plugins/imports install-dynamic && \
   $YARN export-dynamic -- --filter=./dynamic-plugins/wrappers/* && \
-  $YARN copy-dynamic-plugins dist
+  $YARN copy-dynamic-plugins dist && \
+  # insert build time into the build-metadata.json
+  now=$(date -u +%FT%TZ); sed -i packages/app/src/build-metadata.json -r -e "s/\"(Last Commit: )(.+)\"/\"\1\2\", \"Build Time: $now\""
```

Comment:
Same here

Reply:
if versions:metadata could optionally add a second date, then sure. we could use that instead.